packages/react/src/Header/Header.tsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import clsx from 'clsx'
+import { ForwardedRef, forwardRef, HTMLAttributes, ReactNode } from 'react'
+import { Heading } from '../Heading'
+import { Logo, LogoBrand } from '../Logo'
+import { VisuallyHidden } from '../VisuallyHidden'
+
+export interface HeaderProps extends HTMLAttributes<HTMLElement> {
+  logoBrand?: LogoBrand
+  logoLink?: string
```

Comment:
`logoLink` is required, but we could set the default to `"/"`

Reply:
If you set it to a default there is no longer a need to require it right? Or the default wil never "kick in"?